alembic/versions/368b8d1f414d_t04_add_relationships.py

```diff
@@ -0,0 +1,83 @@
+"""T04:Add relationships
+
+Revision ID: 368b8d1f414d
+Revises: 3f13137eb801
+Create Date: 2022-01-25 19:33:35.632681
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.orm import relationship
+
+
+# revision identifiers, used by Alembic.
+revision = '368b8d1f414d'
+down_revision = '3f13137eb801'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+
+    #New columns playlist table
+    op.add_column('playlist', sa.Column('collaborative', sa.BOOLEAN))
+    op.add_column('playlist', sa.Column('public', sa.BOOLEAN))
+    op.add_column('playlist', sa.Column('description', sa.Unicode(200)))
+
+
+    op.create_table(
+        'user_playlist',
+        sa.Column('id_user', sa.INTEGER, primary_key=True),
```

Comment:
I think Spotify is using UUID as Primary Key, is this an internal PK?

Reply:
Let's keep it as an integer for now, work with UUIDs on mysql is a pain in the neck as it is not native at all